doc/rest.md

```diff
@@ -548,7 +556,7 @@ Content-Type: `application/json`
 JSON object, same as in case of [GET Workspace Map](#get-workspace-map).
 
 ### DELETE Workspace Map
-Delete existing map and all associated sources, including map-composition JSON file and map thumbnail.
+Delete existing map and all associated sources, including map-composition JSON file and map thumbnail. [Asynchronous tasks](async-tasks.md) of affected map are aborted.
```

Comment:
'Asynchronous tasks of affected map are aborted.' -> 'Currently running asynchronous tasks of affected map are aborted.'